.travis.yml

```diff
@@ -17,3 +17,12 @@ deploy:
   on:
     repo: m-lab/m-lab.github.io
     branch: master
+before_install:
+  if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    openssl aes-256-cbc -K $encrypted_69b240b11ca3_key -iv $encrypted_69b240b11ca3_iv -in _cf_s3_invalidator.yml.enc -out _cf_s3_invalidator.yml -d;
```

Comment:
Documentation added to both the Travis config and the README.  Merging and crossing fingers that it works as is, but could needs some tweaking... we'll see.